m3/eval/eval_chexpert_class.slurm

```diff
@@ -7,7 +7,7 @@
 #SBATCH --partition=interactive,interactive_singlenode,grizzly,polar,polar2,polar3,polar4
 #SBATCH --exclusive
 #SBATCH --dependency=singleton
-#SBATCH -A healthcareeng_monai
+#SBATCH -A monai
```

Comment:
Can we use `#SBATCH - A <YOUR_SLURM_ACCOUNT>` in these scripts to avoid confusion.

Reply:
ok, I moved it to env var